modules/products/bitbucket/helm.tf

```diff
@@ -6,7 +6,7 @@ resource "helm_release" "bitbucket" {
   repository = local.helm_chart_repository
   chart      = local.product_name
   version    = local.bitbucket_helm_chart_version
-  timeout    = 10 * 60 # autoscaler potentially needs to scale up the cluster
+  timeout    = 120 * 60 # TODO - convert this to variable with default of 10 minutes
```

Comment:
Or maybe 10 minutes for default replica, and for each extra pod add 2 min?

Reply:
I would try to avoid being explicitly smart regarding timeout - there are too many variables that will impact this. Pod resources, EC2 node capacity in the EKS cluster, snapshot restore.

Exposing it as a variable would provide enough flexibility to solve the problem (the dc apps config can then define the config based on the other parameters and actual tests).

Reply:
As far as we won't add this variable (because of time constrain in this project), we may can add timeout when we have more than one replica make some buffer and decrease the chance of failure until we refactor this part in future if is needed. 
Something like:
```
timeout    = (8 + 2 * var.replica_count) * 60  # autoscaler potentially needs to scale up the cluster
```

Reply:
But that doesn't take into account resources or even if the snapshot is restored. Exposing this variable is quite simple I just didn't want to inflate the PR.